config/ftbquests/quests/chapters/tips_and_tricks.snbt

```diff
@@ -1,442 +1,515 @@
+				}
+				{
+					id: "1159FE6CDF262C46"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			x: -1.0d
+			y: 6.0d
+			shape: "diamond"
+			subtitle: "Honey I Shrunk Myself"
+			description: ["Use this item to shrink. Helpful for working on automation and also just overall fun."]
+			id: "7EC8814940C4C3D7"
+			tasks: [{
+				id: "30C938C85BED7956"
+				type: "item"
+				item: {
+					id: "shrink:shrinking_device"
+					Count: 1b
+					tag: { }
+				}
+			}]
+			rewards: [{
+				id: "156D559B0B56BA9A"
+				type: "xp"
+				xp: 10
+			}]
+		}
+		{
+			title: "Comforts"
+			x: 3.5d
+			y: 5.5d
+			subtitle: "ZZZzzz..."
+			description: [
+				"The sleeping bag lets you sleep during the night."
+				""
+				"The hammock lets you sleep during the day."
+				""
+				"Neither will reset your spawn-point."
+			]
+			id: "14C8FC3F19190054"
+			tasks: [
+				{
+					id: "450D00C38F5DB42B"
+					type: "item"
+					item: "comforts:hammock_white"
+				}
+				{
+					id: "76BF5183FA92EF4B"
+					type: "item"
+					item: "comforts:sleeping_bag_white"
+				}
+			]
+			rewards: [
+				{
+					id: "34037270BA530831"
+					type: "item"
+					item: "comforts:rope_and_nail"
+					count: 2
+				}
+				{
+					id: "0CA8B7411485A9D0"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			x: 4.0d
+			y: 5.0d
+			description: [
+				"A quick way to switch between tools."
+				""
+				"Upgrade with belt pouches in an anvil to increase capacity."
+			]
+			id: "378C95C18798D413"
+			tasks: [{
+				id: "7FD9A02EA29669F6"
+				type: "item"
+				item: "toolbelt:belt"
+			}]
+			rewards: [
+				{
+					id: "40BCE4D56FE98018"
+					type: "item"
+					item: "toolbelt:pouch"
+					count: 2
+				}
+				{
+					id: "4E495FC797AED7A4"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			title: "Compact Machines"
+			x: -1.0d
+			y: 5.0d
+			description: [
+				"The Personal Shrinking Device serves as the guide for Compact Machines, as well as the tool needed to teleport into your tiny machine worlds."
+				""
+				"The Compact Machine itself is a small world that you can teleport in and out of to store your farms and machines!"
+			]
+			id: "73F91BF3D424B269"
+			tasks: [
+				{
+					id: "3B227EED4B839733"
+					type: "item"
+					item: "compactmachines:personal_shrinking_device"
+				}
+				{
+					id: "1B529CAA2620B7CC"
+					type: "item"
+					item: "compactmachines:machine_tiny"
+				}
+			]
+			rewards: [
+				{
+					id: "43605B432D1C8172"
+					type: "xp"
+					xp: 10
+				}
+				{
+					id: "16DBB58D967E5AB0"
+					type: "item"
+					item: "compactmachines:wall"
+					count: 8
+				}
+			]
+		}
+		{
+			title: "Building Gadgets"
+			x: -1.5d
+			y: 5.5d
+			shape: "diamond"
+			description: [
+				"Some useful gadgets for building."
+				""
+				"&cDestruction Gadget voids blocks!"
+			]
+			id: "0F75D7B9F4FF2459"
+			tasks: [
+				{
+					id: "358F3A8EA9E5E7E3"
+					type: "item"
+					item: {
+						id: "buildinggadgets:gadget_building"
+						Count: 1b
```

Comment:
sorry for the changelog here - somehow mc formats it as it desires - if you toggle whitespace it's more readable.

<img width="261" alt="CleanShot 2023-01-05 at 05 21 37@2x" src="https://user-
images.githubusercontent.com/1835434/210694438-06b203d3-3140-4057-bb8a-e67602a91275.png">